src/content/docs/iast/troubleshooting/application-stability-issues.mdx

```diff
@@ -0,0 +1,19 @@
+---
+title: Application stability issue
+tags:
+  - IAST
+  - New Relic IAST
+  - Troubleshooting
+metaDescription: Troubleshoot application stability issues in New Relic platform.
+freshnessValidatedDate: 2024-07-16
+---
+
+## Problem
+
+Your application isn't stable during high traffic and you are facing latency issues.
+
+## Solution
+
+Your application may show some latency for some time as part of IAST during high traffic. This should resolve within a few minutes after it finishes the IAST test.
```

Comment:
"the IAST test"....should just be "...finishes IAST"

Reply:
done